docs/cc_to_ako.md

```diff
@@ -0,0 +1,90 @@
+# Cloud connector to AKO migration
+## Overview
+Cloud connector(CC) is not supported from Avi Controller versrion 18.2.10 onwards. To migrate existing workload present in kubernetes cluster from cloud connector to AKO, following two features will be used.
+
+* <b>Namespace-Driven Inclusion/Exclusion of OpenShift/Kubernetes Applications</b>:
+
+    For migration activity,  Exclusion feature allows ingresses/routes from specific namespace(s) to be deleted(excluded) from Avi Controller. For that namespace has to be labelled with same key:value pair as that of exclusion attributes mentioned in cloud. This feature is supported in Cloud Connector based Avi Controller. 
+
+    Details about enabling exclusion feature can be found [here](https://avinetworks.com/docs/18.2/namespace-inclusion-exclusion-in-openshift-kubernetes/)
+
+* <b>Namespace Sync feature</b>
+
+    Namespace sync feature allows K8 objects from specific namespace to be synced with Avi-Controller. For that, namespace has to be labelled with same key:value pair as that labelKey and labelValue mentioned in values.yaml. This feature is supported by AKO from 1.4.1. 
+    
+    Details about this feature is at: [Namespace Sync in AKO](objects.md#namespace-sync-in-ako)
+
+So crux of migration activity to remember is to use same "key:value" pair 
+
+1. as an exclusion attributes in Cloud connector setup.
+2. as a namespace selector in AKO
+3. as a label to namespace whose kubernetes objects needs to be migrated.
+
+This will delete virtual services from CC based Avi-controller and create new virtual services at AKO based Avi-Controller for ingresses, L4 services of that namespace.
+
+## Workflow
+
+This section gives details about steps of migration.
+
+
+![Alt text](images/workflow-cc-to-ako.png?raw=true)
+
+### 1. Preparation for migration
+
+*  Setup a new controller (compatible with AKO version 1.4.1 and above)
+*  Setup a new Vcenter cloud with IPAM.
+*  Setup a DNS Service
+*  Replicate AVI side objects referred by Ingresses/Services as part of AVI_PROXY annotations to new           controller.
+
+### 2. Deploy AKO CRDs
+
+For each AVI_PROXY annotation present in an ingress, create corresponding AKO Http rule/Host rule/ AviInfrasetting rule. 
+Details of these CRDs can be found out [here](crds/overview.md)
+
+If any AVI_PROXY annotation is not supported by these CRD's, then VS which is migrated to AKO will not have same features as that in CloudConnector. 
+
+### 3. AKO Deployment
+
+Deploy AKO (version 1.4 and above) with namespace sync feature enabled.
```

Comment:
I think it'll be good to have a link that describes how to set t he namespace sync feature. `values.md` probably.

Reply:
Screen shot pasted for this section shows how to set namespace sync. Also we have given link of page that explains how to perform namespace sync.